### DISCUSSION TITLE
The progress bars look different now, how do I fix it?

### DISCUSSION THREAD

**Discussion by ySergi** [microsoft/terminal#17124] [Q&A] [answered]
This occurs also with the winget progress bar, as you can see there are small black lines inside the progress bar:
![image](https://github.com/microsoft/terminal/assets/138529583/b2386694-9e3a-4e8c-a532-071458263956)
The Atlas rendering engine is enabled, it always was and the progress bars always looked good. But for some strange reason now they don't look good anymore.

**Comment by lhecker** [comment, marked as answer]
When you say that it doesn't look good, do you mean these thin vertical lines between the █ characters? That issue has recently been addressed in #16729. It'll release in the upcoming Windows Terminal Preview 1.21 version. If you can't wait, please feel free to try out our nightly Canary version: https://aka.ms/terminal-canary-installer

**Reply by ySergi** [reply]
Oh, I thought it was a problem on my side. Nah, it's fine, I'll wait. Thanks for the answer.
